schema/transportation/segment.yaml

```diff
@@ -299,14 +299,21 @@ properties:
       description: >-
         A speed value, i.e. a certain number of distance units
         travelled per unit time.
-      type: array
-      prefixItems:
-        - description: Number of speed units
+      type: object
+      properties:
+        value:
+          description: Number of speed units  
           type: integer
           minimum: 20
-        - description: One speed unit
+        unit:
+          description: One speed unit
           type: string
-          enum: [ "km/h", "mph" ]
+          enum: 
+            - km/h
+            - mph
+      required:
+        - value
+        - unit  
       additionalItems: false
```

Comment:
This should switch to `unevaluatedProperties: false`. The `additionalItems` property only applies to arrays, not objects.